README.md

```diff
@@ -39,6 +39,11 @@ At a minimum, you need to specify `host`, `user`, `password`, `schema` (database
 | 1.0.0.x           | ❌           | ✅          | ✅          | ✅          | ❌          |
 |1.1.0.x            | ❌           | ✅          | ✅          | ✅          | ✅          |
 
+## Version compatibility 
```

Comment:
Can you change the title to dbt dependent package version compatibility

Reply:
Made the changes.